File: reactjs-reducer-hook-json-data/app/components/Product.js
import React from 'react';

const Product = ({ name, description, price }) => (
    <dl>
        <dt>Product Name:</dt>
        <dd>{name}</dd>
        <dt>Product Description:</dt>
        <dd>{description}</dd>
        <dt>Product Price:</dt>
        <dd>{price}</dd>
    </dl>
)

export default Product

File: SalesSystem/SalesSimulator/Models/Product.cs

namespace SalesSimulator.Models
{
    using System.Text.Json.Serialization;
    public sealed class Product
    {
        [JsonPropertyName("id")]
        public int Id { get; set; }
        [JsonPropertyName("seller")]
        public string Seller{get; set;}
        [JsonPropertyName("name")]
        public string Name { get; set; }
        [JsonPropertyName("price")]
        public double Price { get; set; }
        [JsonPropertyName("categories")]
        public string[] Categories { get; set; }
    }
}